Makefile

```diff
@@ -101,6 +104,9 @@ install-ui: build-ui
 install-external-extensions:
 	pip install --upgrade jupyterlab-git==$(GIT_VERSION)
 	jupyter labextension install --no-build @jupyterlab/toc@$(TOC_VERSION)
+	pip install --upgrade nbresuse==$(NBRESUSE_VERSION)
+	jupyter labextension install --no-build jupyterlab-topbar-extension@$(TOPBAR_VERSION)
+	jupyter labextension install --no-build jupyterlab-system-monitor@$(MONITOR_VERSION)
```

Comment:
I would prefer not to depend on a user-specific extension, to avoid the issue that happened with `phosphor`.

Reply:
In that case we will need to write the API as well because `nbresuse` is also a user specific extension